lib/graphql/mutations.js

```diff
@@ -71,18 +71,7 @@ export const addCreateCollectiveMutation = graphql(createCollectiveMutation, {
         pick(tier, ['type', 'name', 'description', 'amount', 'maxQuantity']),
       );
       CollectiveInputType.location = pick(collective.location, ['name', 'address', 'lat', 'long', 'country']);
-      return await mutate({
-        variables: { collective: CollectiveInputType },
-        update: (store, { data: { createCollective } }) => {
```

Comment:
Without this `update`, is the `LoggedInUser` still properly updated with the new membership?

Reply:
I'm assuming we refetch the loggedInUser. This is why I thought we don't need to update the cache.

https://github.com/opencollective/opencollective-frontend/blob/master/components/create-collective/index.js#L82

Reply:
This `update` was probably not working, otherwise the need to add these `refetchLoggedInUser` wouldn't have been there. It was mutating the original data, something the `update` function is not supposed to do.